crates/swift-bridge-ir/src/bridged_type/bridgeable_result.rs

```diff
@@ -430,15 +424,16 @@ impl BuiltInResult {
         if !self.is_custom_result_type() {
```

Comment:
Can we delete this panic since we already have extensive testing for this case

Reply:
Can rename the method to `custom_c_struct_name` to make it more clear that this is only used for custom results.